curriculum/challenges/english/25-front-end-development/quiz-asynchronous-javascript/66edd630f7666cfa54b404d0.md

```diff
@@ -281,175 +297,205 @@ It pauses the execution of the function until the promise is resolved.
 
 #### --text--
 
-What is the purpose of the Geolocation API?
+Which of the following API's is used to provide a way for websites to request the user's location?
 
 #### --distractors--
 
-To manipulate HTML elements.
+Locator API
 
 ---
 
-To create and manage map elements.
+Geo API
 
 ---
 
-To control the device's camera.
+Locate API
 
 #### --answer--
 
-To retrieve the geographical location of a device.
+Geolocation API
 
 ### --question--
 
 #### --text--
 
-Which method of the Geolocation API is used to get the current position of the device?
+Which of the following attributes downloads the script asynchronously but defers the execution of the script until after the HTML document has been fully parsed?
 
 #### --distractors--
 
-`getLocation()`
+`asyncDefer`
 
 ---
 
-`getPosition()`
+`deferred`
 
 ---
 
-`getLatLong()`
+`deferring`
 
 #### --answer--
 
-`getCurrentPosition()`
+`defer`
 
 ### --question--
 
 #### --text--
 
-What parameters can be passed to the `getCurrentPosition()` method?
+What will the following code return? 
+
+```js
+fetch('https://api.example.com/data')
+  .then(response => response.json())
+  .then(data => console.log(data))
```

Comment:
```suggestion
```

Could maybe delete these? The first part does what the answer says. But it's fine as-is too, I think it's clear enough what is being asked.